gcsfs/tests/test_core.py

```diff
@@ -308,19 +308,20 @@ def test_get_put(token_restore):
             assert gcs.cat(TEST_BUCKET+'/temp') == data
 
 
```

Comment:
`parametrize` seemed like a simple way of making sure different protocols work, but I'm happy to make an explicit test for that if that's preferred